src/components/entity/list.vue

```diff
@@ -341,6 +341,8 @@ export default {
 
       const $search = this.searchTerm ? this.searchTerm : undefined;
 
+      this.clearSelectedEntities();
```

Comment:
In the PR description, you wrote:

> Clear selection when refresh request is completed (was happening already because of watcher on odataEntities, added test)

I see that this line is also clearing the selection right before the request is sent. If the watcher clears the selection after the response, am I thinking correctly that the selection is cleared both before and after? Is the selection cleared before the request just to make it clear as soon as possible that clicking the Refresh button clears the selection? Or to phrase the question differently, what are the benefits of clearing the selection both before and after vs. just after? I'm happy for us to keep this line, I just wanted to make sure I understood the reasoning.